js/renovation/ui/scheduler/workspaces/base/date_table/all_day_panel/cell.tsx

```diff
@@ -0,0 +1,32 @@
+import {
+  Component, ComponentBindings, JSXComponent, OneWay,
+} from 'devextreme-generator/component_declaration/common';
+import { getGroupCellClasses } from '../../../utils';
+
+export const viewFunction = (viewModel: AllDayPanelCell) => (
+  <td
+    // eslint-disable-next-line react/jsx-props-no-spreading
+    {...viewModel.restAttributes}
+    className={`${viewModel.classes} ${viewModel.props.className}`}
```

Comment:
Consider moving this concatenation to the `classes` getter to keep view function simple